xcodeproj/internal/bazel_integration_files/calculate_output_groups.py

```diff
@@ -36,9 +51,32 @@ def _calculate_output_group_target_ids(
             # Parse the build-request.json file
             build_request = json.load(f)
 
-        buildable_target_ids = []
+        # Xcode gets "suck" in the `buildFiles` or `build` command for top-level
```

Comment:
Xcode does sometimes suck, but I don't think that's what you meant to say here 🙃 

```suggestion
        # Xcode gets "stuck" in the `buildFiles` or `build` command for top-level
```